src/Features/CSharp/Portable/Snippets/CSharpClassSnippetProvider.cs

```diff
@@ -24,9 +24,9 @@ internal class CSharpClassSnippetProvider : CSharpTypeSnippetProvider
         public CSharpClassSnippetProvider()
         {
         }
-        public override string SnippetIdentifier => "class";
+        public override string Shortcut => "class";
 
-        public override string SnippetDescription => FeaturesResources.class_;
```

Comment:
TBO I preffer `Description` over new `Title`. `Title` is something eye-catching, while this text is displayed on a side of a completion item, so I would probably call it a "small description" rather than a title. If you need to differentiate it from what is displayed outside the completion list, well, it is a _tooltip_